src/Searchable/Aggregator.php

```diff
@@ -41,6 +41,13 @@ abstract class Aggregator implements SearchableCountableContract
      */
     protected $model;
 
+    /**
+     * Map of model relations to load.
```

Comment:
What do you think about this description: `The relationships per model that should be loaded.`.

Reply:
Yes, i'd be happy with that 👍